LCLSGeneral/LCLSGeneral/POUs/Hardware/FB_LED_PWM.TcPOU

```diff
@@ -0,0 +1,56 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<TcPlcObject Version="1.1.0.1" ProductVersion="3.1.4022.18">
+  <POU Name="FB_LED_PWM" Id="{5fae4b24-b1d6-4ac5-a010-9aff4e2893db}" SpecialFunc="None">
```

Comment:
I think we should take a moment to pause and consider if this is a generalized LED_PWM FB or if it's a very specific ELsomethingsomething FB. I think matching the identity is important.